### PR TITLE
Update Docs & Add Test Cases

### PR DESCRIPTION
**Pull request by R-Ramana**
Add Additional Test Cases and updated PPP

### CODE REVIEW

File: src/test/java/seedu/notus/util/parser/ParseArchiveOrUnarchiveNoteCommandTest.java [hunk 1]
```diff
@@ -2,14 +2,177 @@
...
+
+        parser = new ParseArchiveOrUnarchiveNoteCommand(null, false);
+        exception = assertThrows(SystemException.class, () -> parser.parse());
+        expectedMessage = ExceptionType.EXCEPTION_MISSING_MESSAGE_AFTER_COMMAND.toString();
+        actualMessage = exception.getMessage();
+        assertTrue(actualMessage.contains(expectedMessage));
+    }
+
+    @Test
+    void parseCommand_missingTagPrefix_throwException() {
```
Comment (Chongjx) [inline src/test/java/seedu/notus/util/parser/ParseArchiveOrUnarchiveNoteCommandTest.java:84]: Is this the correct function name?

File: src/test/java/seedu/notus/util/parser/ParseListNoteCommandTest.java [hunk 2]
```diff
@@ -2,14 +2,325 @@
...
+        }
+
+        parser = new ParseListNoteCommand(inputString);
+
+        try {
+            command = parser.parse();
+            assertTrue(command instanceof ListNoteCommand);
+        } catch (SystemException exception) {
+            exception.printStackTrace();
+        }
```
Comment (Chongjx) [inline src/test/java/seedu/notus/util/parser/ParseListNoteCommandTest.java:44]: The inputString is not changed? Likwise there are few calls but the inputString wasnt changed.

**Review by Chongjx — APPROVED** [review]
Some things to look out for.

**Review by Nazryl — APPROVED** [review]

File: docs/team/r-ramana.md [hunk 3]
```diff
@@ -39,14 +50,23 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
...
+1. Maintained Issues and Reviewed PR alongside Jun Xiang.
+1. Updating of User/Developer documents that are not feature specific.
+1. Maintained Gantt Chart.
+1. Created a Development Practices Document.
 
 ### Contributions to User Guide
-Created the first draft of the Developer Guide and provided a template for the rest to follow. Also in charge of the overall formatting for the Developer Guide.
+Created the first draft of the User Guide and provided a template for the rest to follow. Also in charge of the overall formatting for the User Guide.
 Filled up the following:
 1. Introduction
```
Comment (Chongjx) [inline docs/team/r-ramana.md:71]: Can link it to the specific segment in the UG and DG

**Review by Chongjx — APPROVED** [review]

File: docs/team/r-ramana.md (hunk 3 above)
Comment (R-Ramana) [reply docs/team/r-ramana.md:71]: Will do in the next round of changes

**Pull request merged**